### ISSUE TITLE
Arrays are merged instead of overwritten

### ISSUE THREAD

**Issue by ArcoMul** [Peerby/node-configuratoror#3]
When a 'parent' config has an array, it's merged with the array defined in the 'child' config.

This way it is not possible to empty arrays.

**Comment by GriffinSauce** [comment]
This seems to work:

config a:
```.js
{
    array: ['a', 'b']
}
```
config b:
```.js
{
    extends: 'a',
    array: null,
}
```

Is that sufficient?

**Comment by ArcoMul** [comment]
Yep, that's how I fixed it now too. What if you want to have the value `['c']` though? Also defining `[]` as value would be nicer than `null` imo.
